Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -258,6 +258,13 @@
     for verdict in verdicts:
         VERDICTS_TO_DBOTSCORE[verdict] = 3
 
+reliability = PARAMS.get('integrationReliability', 'B - Usually reliable')
+
+if DBotScoreReliability.is_valid_type(reliability):
+    reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
```

Comment:
It is a global if I'm not mistaken. please capitalize

Reply:
It is changes in the if, therefore not consider as a Constant as far as I know.